tsconfig.json

```diff
@@ -3,6 +3,7 @@
         "declaration": true,
         "downlevelIteration": true,
         "incremental": true,
+        "lib": ["ES2021.Promise"],
```

Comment:
Do we know if there are any compatibility concerns with this? What is used in Power Query?